reader_controller.py

```
@@ -0,0 +1,26 @@
+import sys
+import re
+from PIL import Image
+from io import BytesIO
+from collections import OrderedDict
+from functools import partial
+
+from metadata_gui_panel import MetadataDialog
+from reader import Comic, SimpleReader
+
+class ReadingController:
+    def __init__(self, comic:dict):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)